Documentation/disaster-recovery.md

```diff
@@ -0,0 +1,112 @@
+# Disaster Recovery
+
+Self-hosted Kubernetes clusters are vulnerable to the following catastrophic
+failure scenarios:
+
+- Loss of all api-servers
+- Loss of all schedulers
+- Loss of all controller-managers
+- Loss of all self-hosted etcd nodes
+
+To minimize the likelihood of any of the these scenarios, production
+self-hosted clusters should always run in a [high-availability
+configuration](https://kubernetes.io/docs/admin/high-availability/).
```

Comment:
I'm on the fence about linking to those docs -- as they're pretty different to how self-hosted HA works (which we need docs for: https://github.com/kubernetes-incubator/bootkube/issues/311). It does touch on some important topics like leader-election, but even then we already have that and all we care about is scaling replica counts (for example).